src/pages/settings/Profile/CustomStatus/SetDatePage.tsx

```diff
@@ -58,31 +65,24 @@ function SetDatePage({translate, customStatus}) {
                 submitButtonText={translate('common.save')}
                 validate={validate}
                 enabledWhenOffline
-                shouldUseDefaultValue
             >
                 <InputWrapper
                     InputComponent={DatePicker}
                     inputID={INPUT_IDS.DATE_TIME}
                     label={translate('statusPage.date')}
                     defaultValue={DateUtils.extractDate(customClearAfter)}
                     minDate={new Date()}
+                    shouldUseDefaultValue
```

Comment:
Let's test that it doesnt break anything

Reply:
Checked manually and it seems that everything is fine